cmd/egress-v4-cni-plugin/cni.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/containernetworking/cni/pkg/skel"
 	"github.com/containernetworking/cni/pkg/types"
-	"github.com/containernetworking/cni/pkg/types/current"
+	current "github.com/containernetworking/cni/pkg/types/100"
```

Comment:
currently our [spec version](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/misc/10-aws.conflist#L2) is 0.4.0, maybe we can just refer to types/040?

Reply:
Ah okay, that's what I was concerned about. Let me update that and check

Reply:
Kept the spec version to 0.4.0